java-frontend/src/main/java/org/sonar/java/se/checks/SECheck.java

```diff
@@ -103,10 +108,20 @@ public void reportIssue(Tree tree, String message, Set<List<JavaFileScannerConte
       if (finalNode.programPoint.syntaxTree() == null) {
         continue;
       }
-      finalNode.getLearnedConstraints().stream()
-        .filter(lc -> lc.getSv().equals(currentVal) && constraintPredicate.test(lc.getConstraint()))
-        .findFirst()
-        .ifPresent(lc -> flow.add(new JavaFileScannerContext.Location("...", finalNode.parent().programPoint.syntaxTree())));
+
+      if (finalNode.getLearnedConstraints().stream()
+        .filter(lc -> lc.getSv().equals(currentVal))
+        .map(lc -> lc.getConstraint())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)